example/SliderExample.tsx

```diff
@@ -3,14 +3,15 @@ import {Text, View, StyleSheet} from 'react-native';
 import Slider, {SliderProps} from '@react-native-community/slider';
 
 const SliderExample = (props: SliderProps) => {
-  const [value, setValue] = useState(0);
+  const [value, setValue] = useState(props.value || 0);
```

Comment:
I noticed that the SliderExample initial value was always 0, even though we pass pass it on a couple examples.